app/models/concerns/wp_post.rb

```diff
@@ -17,6 +17,15 @@ def update_post(json)
   end
 
   module ClassMethods
+
+    #
+    # By default only query on published WpPosts.
+    # Exclude 'draft' or 'pending' statusses.
+    #
+    def default_scope
+      where status: 'publish'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.